portfolio/src/main/webapp/script.js

```diff
@@ -91,3 +91,39 @@ async function deleteComments() {
     });
     getComments();
 }
+
+/** Initiates the meme page */
+async function loadMemePage() {
+    let form = await fetchBlobstoreUrlAndShowForm();
+    let memes = await loadMemes();
```

Comment:
the functions don't look like they return anything, just do `await function()`

Reply:
Ah I ddint know I could do that